### DISCUSSION TITLE
How to get the Quickdraw Badge

### DISCUSSION THREAD

**Discussion by ScremerMemer** [community/community#144773] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

According to this [repository](https://github.com/Thinkright20/Profile-Badges), I can get the quick draw badge by closing a Pull Request or Issue in 5 mins of it being opened. (You can close your own issues), I tried both yet I didn't get the badge, what am I doing wrong, please help.

**Comment by BancoCentraI** [comment]


**Reply by ScremerMemer** [reply]
ChatGPT response, ignored.

**Comment by davevad93** [comment]
Hi @ScremerMemer ,

That's odd because that's exactly what you have to do to get the badge, just open a `pull request` in any repo you own and close it immediately afterwards by clicking on the `Close pull request` button.

**Reply by ScremerMemer** [reply]
Yes idk why it isnt working

**Reply by ScremerMemer** [reply]
Could there be an issue with my settings?

**Reply by davevad93** [reply, marked as answer]
Hey @ScremerMemer , I checked your achievements and you actually earned the `Quickdraw Badge`, go to your [achievements tab](https://github.com/ScremerMemer?tab=achievements) click on the badge and then ***Hide***-***Unhide***. 

![00000](https://github.com/user-attachments/assets/467e5a37-07ca-4e30-8054-c2d8a9359057)

![1111](https://github.com/user-attachments/assets/a3229210-f6e4-4f79-8ec4-1d71371552ea)

It happens sometimes, so always check first your achievements tab.

**Reply by ScremerMemer** [reply]
Yeah, thanks

**Comment by mecodeatlas** [comment]
Hi @ScremerMemer 👋🏾

It looks like @davevad93's response was the answer to your question here. Can you please mark their response as the answer? When you mark a question as an answer, GitHub will highlight the comment and replies to the comment to help visitors quickly find the answer.
![image](https://github.com/user-attachments/assets/b89e7161-c2d1-47e2-a058-242df633e6b2)

**Reply by ScremerMemer** [reply]
I marked the response as the answer a while ago
